ui/app/rif/actions.js

```diff
@@ -25,23 +27,35 @@ function showDomainsDetailPage (data) {
     }
   }
 }
+
+function showAddNewMulticryptoAddressPage (domain, selectedResolverIndex) {
+  return {
+    type: rifActions.SHOW_ADD_NEW_MULTICRYPTO_ADDRESS_PAGE,
+    value: {
```

Comment:
Why using two levels of `value` here? Can we just use only one `value` key?

Reply:
This code was unused, as now the new multicrypt is a modal